spec/comment_spec.rb

```diff
@@ -0,0 +1,11 @@
+# spec/models/post_spec.rb
+
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  describe '#update_comment_counter' do
+    it 'is available' do
+      expect(Comment.new).to respond_to(:update_comment_counter)
+    end
+  end
+end
```

Comment:
- [ ] **[OPT]** I suggest that you refactor this test, as of right now it is just testing that the Comment has a method named `update_comment_counter`. However, if you had an empty method, this will still pass. 🤔 What if instead of testing the implementation details, you test the actual behavior of this method, e.g. checking if after calling it, the post's `comments_counter` increments? This would be a test that provides a lot more confidence! 🤓 🔧